manifest.json

```diff
@@ -21,7 +23,9 @@
     "vtex.shipping-estimate-translator": "2.x",
     "vtex.address-form": "4.x",
     "vtex.profile-form": "3.x",
-    "vtex.pixel-manager": "1.x"
+    "vtex.pixel-manager": "1.x",
+    "vtex.pwa-components": "0.x",
+    "vtex.store-resources": "0.x"
```

Comment:
Way don't you bump this bad-boys versions? :)

https://semver.org/#spec-item-4

Reply:
@salesfelipe I can bump pwa-components to be **1.x**, but store-resources I don't think that I can :eyes:

Reply:
This is just a thing to keep in mind, isn't a must...